src/ECharts/Utils.purs

```diff
@@ -1,6 +1,15 @@
 module ECharts.Utils where
 
+import Prelude
```

Comment:
I suppose it would be nice if formatter functions are live in separate packages, like `Data.Number.Format` and `Data.Date.Format` or something like this. It could help with extraction these funcs to separate package later.